common/static/common/js/components/views/paging_header.js

```diff
@@ -20,7 +20,9 @@
                     end = start + this.collection.length,
                     num_items = this.collection.totalCount,
                     context = {first_index: Math.min(start + 1, end), last_index: end, num_items: num_items};
-                if (end <= 1) {
+                if (this.collection.length === 0) {
```

Comment:
Nit: could we get rid of this extra conditional by making sure `first_index` and `num_items` are always integers (never `NaN`)?